calour/experiment.py

```diff
@@ -274,7 +278,8 @@ def reorder(self, new_order, axis=0, inplace=False):
         elif axis == 1:
             exp.data = exp.data[:, new_order]
             exp.feature_metadata = exp.feature_metadata.iloc[new_order, :]
-
+        else:
+            raise ValueError('axis %s unknown' % axis)
```

Comment:
removed. but didn't manage to add the decorator inside the class.... left as comment the decorator